boards/arm/stm32l562e_dk/stm32l562e_dk_common.dtsi

```diff
@@ -12,11 +12,11 @@
 	leds {
 		compatible = "gpio-leds";
 		red_led_9: led_9 {
-			gpios = <&gpiod 3 GPIO_ACTIVE_HIGH>;
+			gpios = <&gpiod 3 GPIO_ACTIVE_LOW>;
 			label = "User LD9";
 		};
 		green_led_10: led_10 {
-			gpios = <&gpiog 12 GPIO_ACTIVE_HIGH>;
+			gpios = <&gpiog 12 GPIO_ACTIVE_LOW>;
```

Comment:
This would theoretically require a dedicated commit.
At least it would be required to mention this in the commit message

Reply:
Done. Moved the GPIO configuration fix into a separate commit.